django/cantusdb_project/main_app/views/views.py

```diff
@@ -1014,7 +1014,7 @@ def get_queryset(self):
         qs = Feast.objects.all().order_by("name")
         if self.q:
             qs = qs.filter(
-                Q(name__istartswith=self.q) | Q(feast_code__istartswith=self.q)
+                Q(name__icontains=self.q) | Q(feast_code__istartswith=self.q)
```

Comment:
it occurs to me now - we're not displaying the `feast_code` anywhere currently (within the autocomplete widget), are we? We should probably not be displaying `feast_code` matches if the user can't see why it's being matched...